l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -621,13 +621,19 @@ def setDatiDDT(self, invoice, body):
         return True
 
     def _get_prezzo_unitario(self, line):
-        res = line.price_unit
+        if line.quantity < 0:
+            res = -1 * line.price_unit
+        else:
+            res = line.price_unit
+
```

Comment:
Una riga di codice in meno, se non elimini la riga precedente:
```suggestion
        if line.quantity < 0:
            res = -1 * line.price_unit

```